packages/contracts/src/contracts/current/utils/LibBytes/LibBytes.sol

```diff
@@ -0,0 +1,183 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.4.21;
+
+contract LibBytes {
```

Comment:
Do we have test coverage for this library @hysz? I think it would be worthwhile since we are likely to re-use these in many places.

If this is the only change blocking this PR, we could also create an Asana task for it. But it should get done before audits.

Reply:
I think this is really important and just created a set explicit test cases for this library. Thanks for pointing this out 👍